src/index.ts

```diff
@@ -1 +1,4 @@
 export * from './components';
+export * from './kas';
+export * from './kafka';
+export * from './service-registry';
```

Comment:
I don't think we need all this structure. A single folder for the components will work just fine. It's all about giving components proper names.

Reply:
As we are planning to move all the visual components from different modules (kas-ui, kafka-ui, srs-ui etc) , so it would be better to have separate folder for each module and it will easy to find out the components.
Top level components folder is for shared components only.
We already have similar structure for Metrics . we need to move Metrics out side of components folder.
It will be more clear.